cms/static/js/models/xblock_info.js

```diff
@@ -238,7 +248,24 @@ function(Backbone, _, str, ModuleUtils) {
         */
        isEditableOnCourseOutline: function() {
            return this.isSequential() || this.isChapter() || this.isVertical();
-       }
+       },
+
+       /*
+        * Check whether any verification checkpoints are defined in the course.
+        * Verification checkpoints are defined if there exists a user partition
+        * that uses the verification partition scheme.
+        */
+        hasVerifiedCheckpoints: function() {
+            var partitions = this.get("user_partitions") || [];
+
+            for (var i = 0; i < partitions.length; i++) {
+                if (partitions[i].scheme === "verification") {
+                    return true;
+                }
```

Comment:
`return Boolean(_.find(partitions, function(p) { return p.scheme === "verification" }));` will be shorter.